src/Microsoft.OpenApi.Workbench/MainWindow.xaml

```diff
@@ -11,8 +11,24 @@
             <ColumnDefinition/>
             <ColumnDefinition />
         </Grid.ColumnDefinitions>
-        <TextBox x:Name="txtInput" Text="{Binding Input}" VerticalScrollBarVisibility="Auto" Margin="10" TextWrapping="Wrap" AcceptsReturn="True" FontFamily="Consolas" />
-        <DockPanel Grid.Column="1" Margin="0,10" >
+    <DockPanel Grid.Column="0"  >
+      <StackPanel DockPanel.Dock="Top">
+        <Label>Input File:</Label>
+        <TextBox x:Name="txtInputFile"
+               Text="{Binding InputFile}" 
+               Margin="10" 
+               TextWrapping="Wrap" AcceptsReturn="false" FontFamily="Consolas" />
+        <Label>Input Content:</Label>
+      </StackPanel>
+      <TextBox x:Name="txtInput" Text="{Binding Input}" 
+               VerticalScrollBarVisibility="Auto" 
+               Margin="10" 
+               TextWrapping="Wrap" 
+               AcceptsReturn="True" 
+               FontFamily="Consolas"
+               VerticalContentAlignment="Stretch" />
+    </DockPanel>
```

Comment:
Do you mind posting a screenshot?

Reply:
I still don't see the screenshot, but I think it's alright.

---
In reply to: [182167917](https://github.com/Microsoft/OpenAPI.NET/pull/241#discussion_r182167917) [](ancestors = 182167917)

Reply:
I put it in the PR description.  I wasn't sure where else to put it.

---
In reply to: [184121152](https://github.com/Microsoft/OpenAPI.NET/pull/241#discussion_r184121152) [](ancestors = 184121152,182167917)

Reply:
Saw it now. Thanks!

---
In reply to: [184128635](https://github.com/Microsoft/OpenAPI.NET/pull/241#discussion_r184128635) [](ancestors = 184128635,184121152,182167917)